### DISCUSSION TITLE
How insert pictures in CustomTkinter

### DISCUSSION THREAD

**Discussion by CBDdu78** [TomSchimansky/CustomTkinter#1558] [General]
I actually work in project and i would like insert a picture in a frame. I read the documentation but i didn't find any help.
It is possible ?
Thank You

**Comment by avalon60** [comment]
> I actually work in project and i would like insert a picture in a frame. I read the documentation but i didn't find any help. It is possible ? Thank You

Hi.

Try something like this:

```
import customtkinter as ctk
from customtkinter import CTkImage
from PIL import ImageTk, Image

# Create an instance of tkinter window
win = ctk.CTk()

# Define the geometry of the window
win.geometry("500x500")

frame = ctk.CTkFrame(win, width=500, height=500)
frame.pack()
frame.place(anchor='center', relx=0.5, rely=0.5)

# Create an object of tkinter ImageTk
img = Image.open("/home/clive/PycharmProjects/ctk5_theme_builder/assets/images/penguin.png")
img = CTkImage(light_image=Image.open("/home/clive/PycharmProjects/ctk5_theme_builder/assets/images/penguin.png"),
               dark_image=Image.open("/home/clive/PycharmProjects/ctk5_theme_builder/assets/images/penguin.png"),
               size=(500, 500))

# Create a Label Widget to display the text or Image
label = ctk.CTkLabel(frame, text=None, image=img)
label.pack()

win.mainloop()

```
You'll need to change the path to the image, but here is the result:

![image](https://user-images.githubusercontent.com/89534395/235968423-f4a2e482-472a-404e-a042-72f08f71a41f.png)

here is the image that I used:

![penguin](https://user-images.githubusercontent.com/89534395/235968506-2a0406b6-b153-45fa-b88a-8834d8d7053b.png)

If you make the fg_color of the frame cyan, like this:

frame = ctk.CTkFrame(win, width=500, height=500, fg_color='cyan')

You get:

![image](https://user-images.githubusercontent.com/89534395/235969671-7462d863-0546-491d-96a8-292b82624c6b.png)

This  latter example assumes that you have an image with a transparent background.

**Reply by Ryzelz** [reply]
how would you do an image at the back while a frame is on the front like this below.
![image](https://user-images.githubusercontent.com/100330023/236043583-f71f277c-0cde-41ec-9004-1d1874b7ed82.png)

**Reply by avalon60** [reply]
> how would you do an image at the back while a frame is on the front like this below. ![image](https://user-images.githubusercontent.com/100330023/236043583-f71f277c-0cde-41ec-9004-1d1874b7ed82.png)

I haven't tried it, but I'm fairly sure that you should be able to set the fg_color, of the top frame, to "transparent".

**Reply by The-Final-Apex** [reply]
cna i ask as well if i can do the same on a textbox
